### PR TITLE
Added a Circular Density Plot under Structured > Circular menu

### PR DESCRIPTION
**Pull request by HawardKetoyoMsatsi**
@africanmathsinitiative/developers @rdstern @Wycklife @lloyddewit this is ready for review but will require the review after PR #6220 is merged. It implements ggplot options similar to the ones under Describe > Specific > Histogram.

### CODE REVIEW

File: instat/instat.vbproj [hunk 1]
```diff
@@ -358,12 +358,24 @@
...
     </Compile>
     <Compile Include="sdgClimaticNCMPMetadata.vb">
       <SubType>Form</SubType>
     </Compile>
+    <Compile Include="sdgDensityLayerOptions.Designer.vb">
+      <DependentUpon>sdgDensityLayerOptions.vb</DependentUpon>
+    </Compile>
+    <Compile Include="sdgDensityLayerOptions.vb">
+      <SubType>Form</SubType>
+    </Compile>
```
Comment (shadrackkibet) [inline instat/instat.vbproj:378]: First, create an empty form so that it can be merged. You could then proceed with implementation of this subdialog.

Comment (HawardKetoyoMsatsi) [reply instat/instat.vbproj:378]: @shadrackkibet A PR adding the empty dialog to frmMain has just been merged, please have a look.

**Comment by lloyddewit** [comment]
@shadrackkibet I understand that the empty dialog was merged in PR #6220, so I think this PR is ready for review. Please could you review? thanks

**Comment by shadrackkibet** [comment]
> @shadrackkibet I understand that the empty dialog was merged in PR #6220, so I think this PR is ready for review. Please could you review? thanks

What about `sdgCircularDensityPlotsOptions`?

File: instat/instat.vbproj (hunk 1 above)
Comment (shadrackkibet) [reply instat/instat.vbproj:378]: Subdialog, not the main dialog.

**Comment by HawardKetoyoMsatsi** [comment]
@shadrackkibet the two subdialogs sdgCircularDensityPlotsOptions and sdgDensityLayerOptions were not included in PR #6220 they are rather included in this current PR #6241.
Will I have to create a separate PR for the subdialogs too?

**Comment by shadrackkibet** [comment]
> @shadrackkibet the two subdialogs sdgCircularDensityPlotsOptions and sdgDensityLayerOptions were not included in PR #6220 they are rather included in this current PR #6241.
> Will I have to create a separate PR for the subdialogs too?

I notice that you do not need new subdialogs i.e ` sdgCircularDensityPlotsOptions` and `sdgDensityLayerOptions`. You should instead use `sdgLayerOptions` and `sdgPlots`.

**Comment by HawardKetoyoMsatsi** [comment]
I am closing this PR so that I can open another PR where issues with the two dialogs are sorted

**Pull request closed without merging**